deploy/prerequisite/longhorn-iscsi-installation.yaml

```diff
@@ -15,7 +15,6 @@ spec:
       labels:
         app: longhorn-iscsi-installation
     spec:
-      hostNetwork: true
```

Comment:
It looks YES, have you tried? the same question for nfs.

Reply:
Yes, I tried.